packages/common/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "@sovryn/common",
+  "version": "1.0.0",
+  "description": "Library for common helpers.",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "source": "src/index.ts",
+  "types": "src/index.d.ts",
+  "private": false,
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "build": "tsc",
+    "start": "tsc -w",
+    "prepare": "run-s build && cp package.json ./dist/package.json && cp README.md ./dist/README.md",
+    "test": "run-s test:unit test:lint test:build",
+    "test:build": "run-s build",
+    "test:lint": "eslint ."
+  },
+  "dependencies": {
+    "debug": "^4.3.1"
+  },
+  "devDependencies": {
+    "prettier": "^2.0.4",
+    "typescript": "^3.7.5"
```

Add npm-run-all package as dev dependency, otherwise build will fail.
```suggestion
    "npm-run-all": "^4.1.5",
    "prettier": "^2.0.4",
    "typescript": "^3.7.5"
```